app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -1,3 +1,48 @@
-package consensus_message
+package consensusmessage
 
-// todo consensus message watcher
+import (
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
```

Comment:
I would leave the external hedera sdk to carry the name `hedera`, and give an alias to our `HederaClient` something like `hederaClient. The way that you've done it I keep getting confused if you are using something from the sdk or something we've written. cc @Daniel-K-Ivanov @failfmi